docs/modules/listener-operator/pages/installation.adoc

```diff
@@ -2,9 +2,8 @@
 
 There are two ways to run the Stackable Listener Operator:
 
-1. Helm managed Docker container deployment on Kubernetes
-
-2. Build from source
+. Helm managed Docker container deployment on Kubernetes
+. Build from source
```

Comment:
I think building from source it's not appropriate here anymore.

I'm making this suggestion as an alternative but this section should probably rewritten (different PRs for all ops) to include OLM too.

```suggestion
. Using the Helm package manager
. Using stackablectl
```

Reply:
Yes you are right, we need to rewrite this section. I would not change just this single bit, because then the rest of the page doesn't make sense.